index.md

```diff
@@ -2,4 +2,6 @@
 
 Hey there! This is a list with publicly available guides on our company culture and values.
 
+* [Remote communication](remote.md)
```

Comment:
Have you deliberately titled this "Remote communication" as opposed to "Remote work" or "Working remotely" or something like that? The page covers a bit more than communication.

Reply:
After so many rewrites I got lost in my own titles. Should be Remote working